Lib/test/test_compile.py

```diff
@@ -1186,6 +1186,15 @@ def f(x, y, z):
             return a
         self.assertEqual(f("x", "y", "z"), "y")
 
+    def test_variable_dependent(self):
+        # gh-104635: Since the value of b is dependent on the value of a
+        # the first STORE_FAST for a should not be skipped. (e.g POP_TOP).
+        # This test case is added for prevent future regression from aggresive optimization.
```

Comment:
```suggestion
        # This test case is added for prevent potential regression from aggresive optimization.
```

Reply:
If we're tweaking this comment, "for prevent" is also not grammatical English; use "to prevent":

```suggestion
        # This test case is added to prevent potential regression from aggressive optimization.
```